src/uniform_sieve.rs

```diff
@@ -0,0 +1,259 @@
+// `λ(m)`, aka Carmichael's function is the LCM of the λ(p)s for each prime used to compute m. In
+// our case, each λ(p) is simply p-1 because each prime appears once (and we exclude 2).
+// Constant values for common `T`s sized from 32 to 2028:
+// For u32, 9 primes,       m = 0xC0CFD797
+//                          λ(m) = 0x0000D890
+// For U64, 15 primes,      m = 0xE221F97C30E94E1D
+//                          λ(m) = 0x0000000000FCF030
+// For U128, 25 primes,     m = 0x5797D47C51681549D734E4FC4C3EAF7F
+//                          λ(m) = 0x00000000000000000000002DE3CB9560
+// For U256, 43 primes,     m = 0xDBF05B6F5654B3C0F5243551439586889F155887819AED2AC05B93352BE98677
+//                          λ(m) = 0x0000000000000000000000000000000000000000000097262FAE1826B96DBE40
+// For U512, 74 primes,     m = 0x106AA9FB7646FA6EB0813C28C5D5F09F077EC3BA238BFB99C1B631A203E81187233DB117CBC384056EF04659A4A11DE49F7ECB29BADA8F980DECECE92E30C48F
+//                          λ(m) = 0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000468A4F19CEA99B05F593F7F03EBA946B4AF700
+// For U1024, 130 primes,   m = 0x02C85FF870F24BE80F62B1BA6C20BD72B837EFDF121206D87DB56B7D69FA4C021C107C3CA206FE8FA7080EF576EFFC82F9B10F5750656B7794B16AFD70996E91AEF6E0AD15E91B071AC9B24D98B233AD86EE055518E58E56638EF18BAC5C74CB35BBB6E5DAE2783DD1C0CE7DEC4FC70E5186D411DF36368F061AA36011F30179
+//                          λ(m) = 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000009220E37A82CBB6007A9DEE07DE852B1FD11D7594688264F7F40E71355F33B7EBFC100
+// For U2048, 232 primes,   m = 0x2465A7BD85011E1C9E0527929FFF268C82EF7EFA416863BAA5ACDB0971DBA0CCAC3EE4999345029F2CF810B99E406AAC5FCE5DD69D1C717DAEA5D18AB913F456505679BC91C57D46D9888857862B36E2EDE2E473C1F0AB359DA25271AFFE15FF240E299D0B04F4CD0E4D7C0E47B1A7BA007DE89AAE848FD5BDCD7F9815564EB060AE14F19CB50C291F0BBD8ED1C4C7F8FC5FBA51662001939B532D92DAC844A8431D400C832D039F5F900B278A75219C2986140C79045D7759540854C31504DC56F1DF5EEBE7BEE447658B917BF696D6927F2E2428FBEB340E515CB9835D63871BE8BBE09CF13445799F2E67788151571A93B4C1EEE55D1B9072E0B2F5C4607F
+//                          λ(m) = 0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000099F002D0502D70DF7B64150D19B477A781987C56EBCDB7637842794D22D0D022010DC9BA1DB141FC9D07A9587E7A130D9DF6F9B67812800D00
+// TODO(dp): Would love for this to be `const`. Or not here at all. Does it have any value, in some form?
+#[allow(unused)]
+fn calculate_m_and_lambda_m<T>() -> (T, T)
```

Comment:
How long does that take, compared to prime searching? If it's not too long, we can just generate it in the sieve on every call. If it is, could be put in https://doc.rust-lang.org/std/sync/struct.LazyLock.html, but that requires `std`, so will have to be gated behind a feature.

Reply:
I had a thought about moving it to a build.rs and spit out a file on disk with the values. That way it wouldn't bloat the binary. But that sucks in a lot of ways.

Another thought I had was to remove the code completely and add a git commit hash as a comment in the code, providing future readers a way to check how the constants were generated if they really care.